rbac/internal/views.py

```diff
@@ -1687,7 +1689,8 @@ def group_assignments(request, group_uuid):
 def check_inventory(request):
     """POST to check relationship from inventory api."""
     # Parse JSON data from the POST request body
-    req_data = json.loads(request.body)
```

Comment:
Can we put this into a util function so we can use it like `req_data = load_req_body(request.body)`